pkg/gateway/mirror/handler/grpc/handler.go

```diff
@@ -1005,7 +977,7 @@ func (s *server) Insert(ctx context.Context, req *payload.Insert_Request) (loc *
 	}
 
 	var mu sync.Mutex
```

Comment:
Can you add comment about these process as well?

Reply:
Thank you for your comment!
I have added new comment.
https://github.com/vdaas/vald/pull/2204/commits/f9cd0ad2f5a78746354a74689de434bad52dad2a